third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -763,7 +763,6 @@ resource "google_compute_subnetwork" "container_subnetwork" {
 	name                     = "${google_compute_network.container_network.name}"
 	network                  = "${google_compute_network.container_network.name}"
 	ip_cidr_range            = "10.0.36.0/24"
-	location                 = "us-central1"
```

Comment:
Do you mind setting this back to region? It was probably a find/replace gone a little too far.